### PR TITLE
fix text wrap issue on summary

### PR DESCRIPTION
**Pull request by jeremy-babylonchain**

### CODE REVIEW

File: src/app/components/Staking/Form/StakingAmount.tsx [hunk 1]
```diff
@@ -87,12 +87,12 @@ export const StakingAmount: React.FC<StakingAmountProps> = ({
     } else {
       setError("");
       onStakingAmountSatChange(satoshis);
-      setValue(maxDecimals(satoshiToBtc(satoshis), 8).toString());
+      setValue(maxDecimals(satoshiToBtc(satoshis), 6).toString());
```
Comment (vitsalis) [inline src/app/components/Staking/Form/StakingAmount.tsx:90]: Here this should not happen. Bitcoin has 8 decimals, as the lowest unit satosh = 10^(-8) Bitcoin

**Review by vitsalis — CHANGES_REQUESTED** [review]
Why this change?

File: src/app/components/Staking/Form/StakingAmount.tsx (hunk 1 above)
Comment (jeremy-babylonchain) [reply src/app/components/Staking/Form/StakingAmount.tsx:90]: yes we should only set display value, my bad

**Comment by vitsalis** [comment]
In general, why are we performing this update? What issue have we encountered?

**Pull request closed without merging**